doc/classes/CanvasItem.xml

```diff
@@ -657,7 +657,7 @@
 		</signal>
 		<signal name="visibility_changed">
 			<description>
-				Emitted when the visibility (hidden/visible) changes.
+				Emitted when the item's visibility changes, either because its own [member visible] property changed or because the visility of one of its ancestors changed (equivalent to a change in [method is_visible_in_tree]).
```

Comment:
```suggestion
				Emitted when the node's visibility changes, either because its own [member visible] property changed or because the visility of one of its ancestors changed (equivalent to a change in [method is_visible_in_tree]).
```

Reply:
Ideally let's say "node", yes. The concept of "CanvasItem" gets lost the further down you go in the inheritance chain.

Reply:
Thanks for the feedback! Replacing 'item' with 'node' while subsequently talking about the 'visibility' property is also a bit confusing, so I opted for spelling out `[CanvasItem]` instead.